src/include/network/postgres/postgres_protocol_utils.h

```diff
@@ -28,7 +28,7 @@ namespace terrier::network {
       {"is_superuser", "on"},
       {"server_encoding", "UTF8"},
       {"server_version", "9.5devel"},
-      {"session_authorization", "postgres"},
```

Comment:
Was this done to match the default database name in the Catalog?

Reply:
This field actually needs to match the user who is requesting access to the database. In all of our tests, the client is currently sending {User : DefaultDatabaseName, Database : DefaultDatabaseName}, with DefaultDatabaseName currently being set to "terrier" by the catalog.